ush/python/pygfs/task/aero_bmatrix.py

```diff
@@ -0,0 +1,294 @@
+        FileHandler(jedi_fix_list).sync()
+
+        # stage backgrounds
+        logger.info(f"Staging backgrounds prescribed from {self.task_config.AERO_BMATRIX_STAGE_TMPL}")
+        aero_bmat_stage_list = parse_j2yaml(self.task_config.AERO_BMATRIX_STAGE_TMPL, self.task_config)
+        FileHandler(aero_bmat_stage_list).sync()
+
+        # generate convert state YAML file
+        logger.info(f"Generate convert state YAML file: {self.task_config.convertstate_yaml}")
+        self.task_config.convertstate_config = parse_j2yaml(self.task_config.INTERPYAML,
+                                                            self.task_config,
+                                                            searchpath=self.gdasapp_j2tmpl_dir)
+        save_as_yaml(self.task_config.convertstate_config, self.task_config.convertstate_yaml)
+        logger.info(f"Wrote convert state YAML to: {self.task_config.convertstate_yaml}")
+
+        # generate diagb YAML file
+        logger.info(f"Generate bmat YAML file: {self.task_config.bmat_yaml}")
+        self.task_config.bmat_config = parse_j2yaml(self.task_config.BMATYAML,
+                                                    self.task_config,
+                                                    searchpath=self.gdasapp_j2tmpl_dir)
+        save_as_yaml(self.task_config.bmat_config, self.task_config.bmat_yaml)
+        logger.info(f"Wrote bmat YAML to: {self.task_config.bmat_yaml}")
+
+        # generate diffusion parameters YAML file
+        logger.info(f"Generate diffusion YAML file: {self.task_config.diffusion_yaml}")
+        self.task_config.diffusion_config = parse_j2yaml(self.task_config.DIFFUSIONYAML,
+                                                         self.task_config,
+                                                         searchpath=self.gdasapp_j2tmpl_dir)
+        save_as_yaml(self.task_config.diffusion_config, self.task_config.diffusion_yaml)
+        logger.info(f"Wrote diffusion YAML to: {self.task_config.diffusion_yaml}")
+
+        # link executable to run directory
+        self.link_bmatexe()
+        self.link_diffusion_exe()
+        self.link_jediexe()
+
+    @logit(logger)
+    def interpBackground(self) -> None:
+        chdir(self.task_config.DATA)
+
+        exec_cmd = Executable(self.task_config.APRUN_AEROANLGENB)
+        exec_name = os.path.join(self.task_config.DATA, 'gdas.x')
+        exec_cmd.add_default_arg(exec_name)
+        exec_cmd.add_default_arg('fv3jedi')
+        exec_cmd.add_default_arg('convertstate')
+        exec_cmd.add_default_arg(self.task_config.convertstate_yaml)
+
+        try:
+            logger.debug(f"Executing {exec_cmd}")
+            exec_cmd()
+        except OSError:
+            raise OSError(f"Failed to execute {exec_cmd}")
+        except Exception:
+            raise WorkflowException(f"An error occured during execution of {exec_cmd}")
+
+        pass
+
+    @logit(logger)
+    def computeVariance(self) -> None:
```

Comment:
Python style guidelines say method names should be snake case, not camel case.
```suggestion
    def compute_variance(self) -> None:
```